service/src/test/kotlin/no/nav/su/se/bakover/service/kontrollsamtale/KontrollsamtaleServiceImplTest.kt

```diff
@@ -0,0 +1,189 @@
+package no.nav.su.se.bakover.service.kontrollsamtale
+
+import arrow.core.left
+import arrow.core.right
+import io.kotest.matchers.shouldBe
+import no.nav.su.se.bakover.client.person.MicrosoftGraphApiOppslag
+import no.nav.su.se.bakover.client.person.MicrosoftGraphApiOppslagFeil
+import no.nav.su.se.bakover.common.persistence.SessionFactory
+import no.nav.su.se.bakover.domain.AktørId
+import no.nav.su.se.bakover.domain.Fnr
+import no.nav.su.se.bakover.domain.Ident
+import no.nav.su.se.bakover.domain.NavIdentBruker
+import no.nav.su.se.bakover.domain.Person
+import no.nav.su.se.bakover.domain.Sak
+import no.nav.su.se.bakover.domain.Saksnummer
+import no.nav.su.se.bakover.domain.oppgave.OppgaveFeil
+import no.nav.su.se.bakover.domain.person.KunneIkkeHentePerson
+import no.nav.su.se.bakover.service.brev.BrevService
+import no.nav.su.se.bakover.service.brev.KunneIkkeLageBrev
+import no.nav.su.se.bakover.service.oppgave.OppgaveService
+import no.nav.su.se.bakover.service.person.PersonService
+import no.nav.su.se.bakover.service.sak.FantIkkeSak
+import no.nav.su.se.bakover.service.sak.SakService
+import no.nav.su.se.bakover.test.TestSessionFactory
+import no.nav.su.se.bakover.test.fixedTidspunkt
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.any
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.mock
+import java.time.Clock
+import java.util.UUID
+
+internal class KontrollsamtaleServiceImplTest {
+
+    val saksbehandler: NavIdentBruker = NavIdentBruker.Saksbehandler("Z999999")
+    val sakId: UUID = UUID.randomUUID()
+    val fnr = Fnr(fnr = "12345678901")
+    val person = Person(
+        ident = Ident(
+            fnr = fnr,
+            aktørId = AktørId(aktørId = "123"),
+        ),
+        navn = Person.Navn(fornavn = "Tore", mellomnavn = null, etternavn = "Strømøy"),
+    )
+    val sak = Sak(
+        id = UUID.randomUUID(),
+        saksnummer = Saksnummer(9999),
+        opprettet = fixedTidspunkt,
+        fnr = fnr,
+        søknader = listOf(),
+        søknadsbehandlinger = listOf(),
+        utbetalinger = listOf(),
+        revurderinger = listOf(),
+        vedtakListe = listOf(),
+    )
+
+    @Test
+    fun `feiler hvis vi ikke finner sak`() {
+        val sakService = mock<SakService> {
```

Comment:
Personlig foretrekker jeg å inline mocks som er helt ubetydelig og ikke referes til noen steder bare for redusere litt kode. Siden du også har tatt det bryet med å wrappe dette inn i et objekt for "service og mocks" ville dette blitt enda mer "kondensert" hvis man inliner mocks og bruker "service and mocks" objektet sine variabler for å gjøre f.eks `verify()` eller `verifyNoMoreInteractions()`.